libs/partners/mongodb/tests/integration_tests/test_vectorstores.py

```diff
@@ -65,6 +103,15 @@ def setup(self) -> None:
         # delete all the documents in the collection
         collection.delete_many({})  # type: ignore[index]
 
+        # delete all indexes on index collection name
+        index_collection = get_collection(INDEX_DB_NAME, INDEX_COLLECTION_NAME)
+        index_collection.delete_many({})
+        try:
+            drop_vector_search_index(index_collection, INDEX_CREATION_NAME)
+        except OperationFailure:
+            # An ongoing drop request was made
```

Comment:
Is there a specific error code we can ignore here (instead of all errors)?

Reply:
Good idea. For posterity, I'll capture it here.
```
pymongo.errors.OperationFailure: Index already requested to be deleted, full error: {'ok': 0.0, 'errmsg': 'Index already requested to be deleted', 'code': 20, 'codeName': 'IllegalOperation', '$clusterTime': {'clusterTime': Timestamp(1710987671, 1), 'signature': {'hash': b'\x04N\x88\xcc7\x05x@\x10xj\xa2\xe7\x15,\xe2\xc7\x16\x04\xfb', 'keyId': 7292037883655880706}}, 'operationTime': Timestamp(1710987671, 1)}
```

Reply:
Eh that error seems to generic to rely on, ignoring all errors here seems okay. Can you open a SERVER ticket to make this case return either a stable error code or ok:1?

Reply:
https://jira.mongodb.org/browse/SERVER-88600